styles/FavoritePin.module.css

```diff
@@ -0,0 +1,26 @@
+.pin {
+  will-change: opacity;
+  opacity: 0;
+  text-decoration: none;
+
+  position: absolute;
+  left: -2rem;
```

Comment:
Why `-2rem`? isn't `-1.5rem` suitable for most cases?

Reply:
It actually looks better